src/i18n/en/index.ts

```diff
@@ -0,0 +1,93 @@
+// import type { BaseTranslation } from '../i18n-types';
+
+// const en: BaseTranslation = {
```

Comment:
Why the comments?

Reply:
so initially I was unable to use the JSON files with the typesafe but for testing purposes, I commented on the objects and tried to import the JSON file which worked successfully.

Reply:
it will be removed today.